### ISSUE TITLE
Installation check of ResourceSpace gives warnings

### ISSUE THREAD

**Issue by nvanderperren** [VlaamseKunstcollectie/Imagehub-Box#7]
The PHP.INI-file should be modified. Is it something that needs to be done during configuration or is it supposed to be done during the setup of the box? In the README it's not mentioned that we should ssh into the vagrant box to modify this file.

This are the warnings:

```
PHP.INI value for 'post_max_size': 32M
(WARNING: should be 100M or greater)

PHP.INI value for 'upload_max_filesize': 64M
(WARNING: should be 100M or greater)
```

**Comment by Hobbesball** [comment]
since the Imagehub Box doesn't install Resourcespace but merely gives it a port to run on, this isn't something we can configure during setup I'm afraid. If we did want to do this automatically, I think we would have to write an ansible role from scratch, and I don't like ResourceSpace enough to do that. I think the best course of action here is to add to the README how to change the ini file so these warnings don't show up. If you know how and want to add it to the documentation, please do! Thanks for noting this 💯

**Comment by nvanderperren** [comment]
ok! will add it.

**Comment by netsensei** [comment]
@nvanderperren 

The configuration files are automatically generated through Ansible and shouldn't be changed directly by SSH'ing in the box. Provisioning through Ansible replaces this.

The solution here is to do this:

**Override the variables for the geerlingguy.php role.**

1. Open `ansible/roles/external/geerlingguy.php/defaults/main.yml`
2. Copy the relevant variables from this file. These are

```
php_post_max_size: "32M"
php_upload_max_filesize: "64M"
```
3. Open `ansible/group_vars/all/php.yml` This file contains all the overriden variables.
4. Paste in the variables and change them accordingly and save the file:

```
php_post_max_size: "100M"
php_upload_max_filesize: "100M"
```

**Apply the new changes to your installation.**

Now, you need to apply the changes to the Vagrant box you've got currently running.
```
$ cd ~/Machines/Imagehub-Box # root path where you installed Imagehub Box
$ vagrant up # make sure your box is up and running
$ cd ansible/plays
$ ansible-playbook -i ../imagehub imagehub.yml -t php -v
```
The ansible-playbook command will run the "geerlingguy.php" role, redo the PHP installation and while doing so apply all the changed configuration settings.

You shouldn't have to restart your box or anything. Reloading the page or POST'ing again should now work without problems.

**Making the change stick**

The change you made will only exist in your currently active box. If you do a `vagrant destroy` you will lose that change and you will have to run ansible-playbook again to apply that change again. Which is annoying. So, run Packer and build the box from scratch again:

````
$ cd ~/Machines/Imagehub-Box
$ vagrant destroy
$ vagrant box remove imagehubbox/bionic64  # (if it can't find the box, do a vagrant box list)
````

Open up Virtualbox and remove all references to packer-imagehubbox

```
$ cd packer
$ packer build imagehub.json
$ cd ..
$ vagrant up
````

It should now install the vagrant box with the changes.

**Make PR**

If you do make a change to the variables in the Ansible folder, please push them as a PR. Thanks!

**Comment by nvanderperren** [comment]
thanks @netsensei for pointing that out. I haven't worked with Ansible yet, so this really helps.

**Comment by netsensei** [comment]
@nvanderperren Top!

Ansible is a great tool for provisioning Vagrant boxes, but also for managing (clusters of) VPS'es. So, I recommend looking into that.

Presently, everyone who wants to install Imagehub-Box (or Datahub-Box for that matter) needs to build their own box file via Packer. Although this is, theoretically, a one time operation, it's time consuming and not that practical.

Normally, Vagrant would download a Vagrant Box from Vagrant Cloud (https://app.vagrantup.com) during `vagrant up` instead of you having to build your own box file from scratch first. So, I've created an account and I've already pushed 2 ready-made boxes for other projects:

https://app.vagrantup.com/vlaamsekunstcollectie

When the last kinks have been worked out, we could create a stable box file for this project too, upload that to Vagrant Cloud and use that in the future instead of having to build a new box.

So, give me a heads up as soon as you feel the box is stable enough in the next few days and you don't experience any more major issues.
